src/components/App/App.js

```diff
@@ -25,7 +25,7 @@ class App extends Component {
         }
       })
       .then((data) => {
-        this.setState({ allMovies: data.movies });
+        this.setState({ allMovies: data.movies, isLoading: false});
```

Comment:
Nice use of a single setState() call to update multiple pieces of state at once!